cal_ratio_trainer/common/column_names.py

```diff
@@ -75,17 +74,26 @@ def _generate_col_mseg_names(index: int) -> List[str]:
 # list of cluster, track, and mseg names
 col_cluster_track_mseg_names = col_cluster_names + col_track_names + col_mseg_names
 
-
-col_jet_names = ["jet_pt", "jet_eta", "jet_phi"]
+col_jet_names = [
+    "jet_pt",
+    "jet_eta",
+    "jet_phi",
+    "jet_isClean_LooseBadLLP",
```

Comment:
Why do we need this now? That isn't input into the training, is it?

Reply:
Very true. It's gone now 🗡️